### PR TITLE
Add how-to for multiscale datasets

### PR DESCRIPTION
**Pull request by btbest**
Docs for loading multiscale datasets in ilastik

### CODE REVIEW

File: documentation/basics/dataselection.md [hunk 1]
```diff
@@ -96,6 +96,40 @@ the names of the files that will be imported as an image stack.
...
+When navigating the dataset, remember that downloading the image data can take some time.
+It is recommended to use the lowest resolution possible for exploration.
+At larger resolutions, zoom in to the area of interest, as suggested in the [performance tips][performance-tips] section.
+This way, ilastik can avoid loading data outside the viewer area.
+
+Once you continue to the next step in the workflow, your choice of scale will be locked for all multiscale entries in the table.
+This is because ilastik can currently not handle changing pixel dimensions once a dataset is loaded into the workflow beyond "Input Data".
+If you want to inspect a different scale later, you can use "Add new" and paste the same address again.
+
+Multiscale datasets are currently supported in the formats OME-Zarr (URLs containing `.zarr`) and Neuroglancer Precomputed (URLs starting with `precomputed://`).
```
Comment (k-dominik) [inline documentation/basics/dataselection.md:130]: I would move this up to directly after the `## Loading multiscale data...` heading - this is the information I think users might care about first.

File: documentation/basics/dataselection.md [hunk 2]
```diff
@@ -120,6 +154,8 @@ Widely used image formats such as
 can be imported directly into a project, as well as NumPy binary array
 files with extension `npy`.
 
+OME-Zarr and Neuroglancer Precomputed are only supported for [multiscale](#multiscale) datasets.
```
Comment (k-dominik) [inline documentation/basics/dataselection.md:157]: I'm getting hung up on the "only" here. It sounds like somehow the support for these files is worse than for the ones mentioned above, but that is not the case. Could you maybe add a new heading "Multiscale data formats" (like "Widely used image formats such as..." and list them, maybe with a hint that "Add multiscale dataset" has to be used to load those.

**Review by k-dominik — APPROVED** [review]
great docs! Level of detail feels "just right" :)

**Comment by btbest** [comment]
Ready to be merged once the feature is out in beta.

**Pull request merged**